libs/schemas/src/events/index.ts

```diff
@@ -1,16 +1,18 @@
 import { z } from 'zod';
+import * as chainEvents from './chain-event.schemas';
```

Comment:
Do we have conflicting names that require us to use a namespace here?

Reply:
Not really. I just tried to use the file name just in case we wanted to have these events segregated, but it can be merged into events